packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -218,3 +217,14 @@ jobs:
           name: playwright-report-$\{{ matrix.GRAFANA_IMAGE.NAME }}-v$\{{ matrix.GRAFANA_IMAGE.VERSION }}-$\{{github.run_id}}
           path: playwright-report/
           retention-days: 5
+
+{{#if_eq orgName "grafana"}}
+      # Repos in the grafana Github org can publish the report to GCS so that they can be browsed without having to download them.
+      # This will make the report public on the Internet, so beware to skip this step if the report contains sensitive information.
```

Comment:
What sort of sensitive information could this expose? Could having it as a default be rather risky?

Reply:
hmm, I'm starting to think that a better approach would be to upload artifacts to github instead (if we want to enable this functionality). Then it would work for all users regarding if it is within the grafana org or not.

Reply:
uploading to github would not give you a possibility to link to it - you would have to download the report and run `npx playwright show-report`

uploading to GCS gives you a way to share a link with someone who doesnt want to download unzip and run report locally - so kind of a convenience feature.

Reply:
> What sort of sensitive information could this expose? Could having it as a default be rather risky?

Same as any e2e test report more or less - it contains your browser state, screenshots of the execution, etc. and if you have something sensitive in there it will be in the report. Since it is quite impossible to automatically know about what could theoretically be sensitive on the screen there is no automated solution to this - https://github.com/microsoft/playwright/issues/19992 

Hence the guidance in the comments above the action - if you are for some reason using real credentials or have sensitive information in the flow - you have to handle it yourself or not publish it to a public bucket.

Reply:
Yep, and the same security concern applies when uploading artifacts to GH as the report becomes public on the Internet. Playwright docs have [example workflows](https://playwright.dev/docs/ci-intro#setting-up-github-actions) where reports are being uploaded to GH artifacts, bit they don't mention security concerns.

Reply:
I'm rather safe than sorry - let's comment out the upload artifacts step.